benchmarks/benchmarks/interpolate.py

```diff
@@ -83,7 +85,32 @@ def setup(self, n_grids, method):
     def time_evaluation(self, n_grids, method):
         interpolate.griddata(self.points, self.values, (self.grid_x, self.grid_y),
                              method=method)
+        
+class GridDataPeakMem(Benchmark):
+
+    def setup(self):
+        shape = (7395, 6408)
+        num_nonzero = 488686
```

Comment:
Is there any logic behind how these numbers were chosen?

Reply:
The `num_nonzeros` is taken from the issue this pull request solves and the `shape` is half the size from the one used in the issue. I did this so that the benchmark takes less time to run. It still showcases the decrease in memory used, though.
I can alter the shape so that it matches the issue's, if you'd prefer it that way.

Reply:
No, that's ok. _Maybe_ add a comment to reference the issue so that future devs know where these numbers come from, but it's not that important.